nginx/nginx.conf

```diff
@@ -17,13 +16,18 @@ http {
   keepalive_timeout 15;
   types_hash_max_size 2048;
   client_max_body_size 20M;
-  include /etc/nginx/mime.types;
-  default_type application/octet-stream;
-  access_log /var/log/nginx/access.log;
-  error_log /var/log/nginx/error.log;
+  open_file_cache max=100;
   gzip on;
   gzip_disable "msie6";
+
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+  ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS';
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
   include /etc/nginx/conf.d/*.conf;
   include /etc/nginx/sites-available/*;
```

Comment:
I had to manually create the symlinks between `/etc/nginx/sites-available/*.conf` and `/etc/nginx/sites-enabled/*.conf`should we do it in the Dockerfile instead?

Reply:
@0x13a sites-enabled is not currently used in the nginx image. Take a look at the https://github.com/Osedea/nodock/blob/master/nginx/scripts/build-nginx.sh script for how the current configs are added to sites-available.